src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -499,8 +490,9 @@ public void traverse() {
             final long variantContextBufferSize = vcfBufferSizePerSample * sampleToReaderMap.size();
             final GenomicsDBImportConfiguration.ImportConfiguration importConfiguration =
                     createImportConfiguration(workspace, GenomicsDBConstants.DEFAULT_ARRAY_NAME,
-                                              variantContextBufferSize, segmentSize,
-                                              i, (i+updatedBatchSize-1));
+                            variantContextBufferSize, segmentSize,
+                            i, (i+updatedBatchSize-1),
+                            (batchCount == 1)); //Fail if array exists and this is the first batch
```

Comment:
Fix indentation here (there may be literal tabs here that need converting to spaces).

Reply:
done